wallet/udb/addressdb.go

```diff
@@ -1244,6 +1246,7 @@ func deletePrivateKeys(ns walletdb.ReadWriteBucket, dbVersion uint32) error {
 		// Deserialize the account row first to determine the type.
 		row, err := deserializeAccountRow(k, v)
 		if err != nil {
+			c.Close()
```

Comment:
What are the rules regarding closing cursors?  Why is this closed here instead of deferring the close as other cursors above? Is it not permitted to open multiple cursors at the same time?

Reply:
Only one cursor can be opened per transaction and should be close when committing the transaction.

I've documented the rules.